src/python/grapl_analyzerlib/grapl_analyzerlib/query_gen.py

```diff
@@ -184,15 +192,10 @@ def into_query_block(
     else:
         filter = ""
 
-    # Grab the properties
-    always = {"uid", "dgraph.type", "node_key"}
-
     tabs = "\t" * 3 + "\t" * depth
```

Comment:
unrelated to your diff, just some extreme early optimizations:

a) couldn't `depth` just be defaulted to `depth = 0` instead of None
b) i have literally no idea but, if `into_query_block` is a relatively common call - would it make sense to dynamic-programming-cache these depth strings

Reply:
also `always` is never gonna change
1. make it module level with a better name
2. make it a tuple not a set (tuples are immutable)